.github/workflows/prune-dist.yml

```diff
@@ -10,10 +10,12 @@ jobs:
   run:
     runs-on: ubuntu-latest
     steps:
+      - name: Clone repository
+        uses: actions/checkout@v2
```

Comment:
Is this just to get `git rev-parse HEAD` below?  I feel like you could get that a lot easier than cloning everything...

Reply:
Indeed. Replaced with `git ls-remote`.